Person-CRUD/Person.Data/PersonRepository.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Person_CRUD.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data
+{
+    public class PersonRepository : IPersonRepository
+    {
+        private readonly CompanyContext _db;
+
+        public PersonRepository(CompanyContext db)
+        {
+            _db = db;
+        }
+
+        public async Task<List<Person>> GetAll()
+        {
+            return await _db.Persons.ToListAsync();
+        }
+        public async Task<bool> CreatePerson(Person person) {
+            _db.Persons.Add(person);
+            _db.SaveChanges();
+            return true;
+        }
+        public async Task<Person> GetById(int id)
+        {
+            return await _db.Persons.FindAsync(id);
+        }
+
+        public async Task<bool> Delete(int id)
+        {
+            var data = await _db.Persons.FindAsync(id);
+            _db.Persons.Remove(data);
+            _db.SaveChanges();
+            return true;
+        }
+
+        public async Task<bool> UpdatePerson(Person person)
+        {
+            var data = await  _db.Persons.Where(x => x.Id == person.Id).FirstOrDefaultAsync();
```

Comment:
Why you didn't use the FindAsync method to find the person by ID, as you are using in GetById method